<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">LicenseSpring SSO C++ SDK Sample</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="ActivationWidget" name="activationScreen"/>
      <widget class="BrowserWidget" name="browserScreen"/>
      <widget class="LicenseInfoWidget" name="licenseScreen"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ActivationWidget</class>
   <extends>QWidget</extends>
   <header>ActivationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BrowserWidget</class>
   <extends>QWidget</extends>
   <header>BrowserWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LicenseInfoWidget</class>
   <extends>QWidget</extends>
   <header>LicenseInfoWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
